app/src/main/java/org/fossasia/openevent/app/MainActivity.java

```diff
@@ -1,16 +0,0 @@
-package org.fossasia.openevent.app;
-
-import android.app.Application;
-import android.content.Context;
-import android.content.SharedPreferences;
-
-import org.fossasia.openevent.app.Utils.Constants;
-
-public class MainActivity extends Application {
-
-    public String getToken(){
-        SharedPreferences sharedPreferences = getSharedPreferences(Constants.fossPrefs, Context.MODE_PRIVATE);
-        String token = sharedPreferences.getString(Constants.sharedPrefsToken,"null");
```

Comment:
please change "Constants.fossPrefs" to "Constants.FOSS_PREFS" and "Constants.sharedPrefsToken" to "Constants.SHARED_PREFS_TOKEN"